index.d.ts

```diff
@@ -0,0 +1,5 @@
+type anyObject = {
+    [key: string]: any,
```

Comment:
Might be `string` and `number` instead of `any`?

```suggestion
    [key: string]: string | number,
```